FIX: Update mouvementstock.class.php

prietojc opened the pull request:
If first character changes to upper case, some external modules don't show its origin in mouvements.

prietojc committed:
FIX: Update mouvementstock.class.php

If first character change to upper case, some external modules don't show its origin in mouvements.

eldy commented on htdocs/product/stock/class/mouvementstock.class.php:
```diff
@@ -975,7 +975,7 @@ public function get_origin($origin_id, $origin_type)
 					// Separate originetype with "@" : left part is class name, right part is module name
 					$origin_type_array = explode('@', $origin_type);
-					$classname = ucfirst($origin_type_array[0]);
+					$classname = $origin_type_array[0];
```
Class name should start with uppercase.
Contact, Warehouse, User, ...
This may generates a lot of trouble if classname is only  contact, warehouse, user...

prietojc replied:
All right I agree,   but uppercase is already done on line 983...
Module name and URL use to be in lowercase.
I propose a new commint more simple

prietojc committed:
Update mouvementstock.class.php

More elegant.
We don't need to change to upper case (line 978) and then to lower case (line 980). It is only necessary when creating a new class (line 983)
The module name and url are in lower case.
